odl/tomo/geometry/parallel.py

```diff
@@ -713,31 +848,71 @@ def det_axes_init(self):
         return self.detector.axes
 
     def det_axes(self, angles):
-        """Return the detector axes tuple at ``angle``."""
+        """Return the detector axes tuple at ``angles``.
+
+        Parameters
+        ----------
+        angles : `array-like`
+            Euler angles in radians describing the rotation of the detector.
+            If multiple angle pairs (or triplets) are given, they must be
```

Comment:
The above proposed fix would solve this so users can give carthesian products, as they would likely expect that they could.